airflow/contrib/operators/ssh_operator.py

```diff
@@ -94,28 +93,30 @@ def execute(self, context):
             stdin.close()
             channel.shutdown_write()
 
-            agg_stdout=b''
-            agg_stderr=b''
+            agg_stdout = b''
+            agg_stderr = b''
+
+            # capture any initial output in case channel is closed already
+            if len(stdout.channel.in_buffer) > 0:
```

Comment:
Could you store `len(stdout.channel.in_buffer)` in a variable so you can reuse it :-)

Reply:
Will amend, debated about this at the time